packages/vitest/src/types/browser.ts

```diff
@@ -53,6 +54,13 @@ export interface BrowserConfigOptions {
    */
   api?: ApiConfig | number
 
+  /**
+   * When running on browser, enable this flag to show the result.
+   *
+   * @default false
+   */
+  enableUI?: boolean
```

Comment:
@userquin  Do we actually need this? I don't have any strong preference over having or not having it as an option, just asking. Can we show it without an option?

Reply:
You can run test in the ui and those test may not have ui, check test/browser

Reply:
It is read in the ui when loading the configuration, it is used to show the tab in the details panel

Reply:
@Aslemammad 

![imagen](https://github.com/vitest-dev/vitest/assets/6311119/59b74aa9-f6ce-4275-a221-66d80c5a104a)

Reply:
I see, That's better, thanks.

Reply:
I think the use case when you run it in the browser without a UI is not so popular. `@vitest/browser` is primarily to allow component testing. examples/browser is just a test to make sure everything works seamlessly